components/release/base/cronjobs/remove-internal-requests.yaml

```diff
@@ -31,7 +31,8 @@ spec:
                   YD=$(date -d 'yesterday' +%s)
                   kubectl get internalrequests --all-namespaces \
                   --sort-by=.status.completionTime \
-                  --template '{{range .items}}{{.metadata.name}}{{"\t"}}{{.metadata.namespace}}{{"\t"}}{{.status.completionTime}}{{"\n"}}{{end}}' > $KUBECTL_OUTPUT
+                  --template '{{range .items}}{{.metadata.name}}{{"\t"}}{{.metadata.namespace}}{{"\t"}}{{.status.completionTime}}{{"\n"}}{{end}}' \
+                    | grep -E "[0-9]{4}((-?)[0-9]{2})+T.*Z$" > $KUBECTL_OUTPUT
```

Comment:
I'm just a little worried that if the date format changes a little, it will not work, whereas deleting lines with `<no value>` would be more straightforward. But maybe it's acceptable - worst case we won't be deleting stuff and we hit the quota again.

Reply:
I had the very same thought but the opposite way. Eventually they change the "no value" string to something else. I guess both options have some risk.

Reply:
I guess the completion time is probably generated by us (the controller) and so the date format should stay like this, so hopefully it's ok.